### ISSUE TITLE
Sometimes, new records are created when old ones should be updated instead

### ISSUE THREAD

**Issue by avollmerhaus** [avollmerhaus/do-update-fqdn#1]
It has happened 2 times already and I have not investigated yet.
Logs:
```
2019-05-24 01:42:20,334 do-update-fqdn: set A record for hostname redacted in domain redacted to 10.252.0.4
2019-05-24 01:42:21,178 do-update-fqdn: no A records found in example.com for hostname somehost, creating
2019-05-24 01:42:22,069 do-update-fqdn: set AAAA record for hostname somehost in domain example.com to fd00:2019::1002
2019-05-24 01:42:22,666 do-update-fqdn: no AAAA records found in example.com for hostname somehost, creating
```

Records afterwards:
```
drill AAAA somehost.example.com @ns1.digitalocean.com
;; ->>HEADER<<- opcode: QUERY, rcode: NOERROR, id: 5617
;; flags: qr aa rd ; QUERY: 1, ANSWER: 2, AUTHORITY: 0, ADDITIONAL: 0 
;; QUESTION SECTION:
;; somehost.example.com.    IN  AAAA

;; ANSWER SECTION:
somehost.example.com.   1800    IN  AAAA    fd00:2019::1010
somehost.example.com.   1800    IN  AAAA    fd00:2019::1002
```

Expected results:
There should only ever be one record for somehost.example.com, the script should create or update that record instead of creating multiple ones.

**Comment by avollmerhaus** [comment]
The problem is that the DO API output is paginated, see https://www.digitalocean.com/community/questions/why-v2-sizes-doesn-t-returns-all-available-sizes

**Comment by avollmerhaus** [comment]
fixed in 1221246ff277cbcb070712d11d7689c538e5380f, formatted commit message wrong.
